docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -105,6 +105,52 @@ include::../../../../extensions/agroal/deployment/src/test/resources/application
 
 There are other configuration options, detailed below.
 
+=== JDBC URL configurations
+
+As noted above, {project-name} currently supports four databases.
+Each of those databases contains different JDBC URL configuration options.
+Going into each of those options is beyond the scope of this document, but it give an overview of each database URL and link to the official documentation.
+
+==== H2
+
+`jdbc:h2:{ {.|mem:}[name] | [file:]fileName | {tcp|ssl}:[//]server[:port][,server2[:port]]/name }[;key=value...]`
+
+H2 is an embedded database.
+It can run as a server, based on a file, or live completely in memory.
+All of these options are available as listed above.
+You can find more information, visit the https://h2database.com/html/features.html?highlight=url&search=url#database_url[official documentation].
+
+==== PostgreSQL
+
+PostgreSQL only runs as a server, as do the rest of the databases below.
+As such, you must specify connection details, or use the defaults.
+
+`jdbc:postgresql:[//][host][:port][/database][?key=value...]`
+
+Defaults for the different parts are as follows:
+
+`host`:: localhost
+`port`:: 5432
+`database`:: same name as the user
+
+The https://jdbc.postgresql.org/documentation/head/connect.html[official docs] go into more detail and list optional parameters as well.
+
+==== MySQL/MariaDB
+
+`jdbc:mysql[:loadbalance|:replication]://[host][/database][?properties]`
+
+MySQL/MariaDB allow for setting the user name and password as part of the host.
```

Comment:
We don't really want that. User and password should be set as Agroal properties. So I would remove that sentence.